### PR TITLE
Add Freespee to Partner Directory

### PR DESCRIPTION
**Pull request by stewsmith**

### CODE REVIEW

**Comment by clarissabee** [comment]
@julieritchie Please review to add to PD! I've e-mailed you their logo as well. Thanks!

**Comment by dtothefp** [comment]
@julieritchie what's the status on this?

**Comment by kellyfinn** [comment]
This is ready @julieritchie @dtothefp @stewsmith

File: website/partners/technology/freespee/index.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: partners.hbs
+title: "Freespee"
+integrations: true
+priority: 53
+summary: "Track phone calls as an offline conversion using Freespee."
+logo: "https://s3.amazonaws.com/cloud_cannon_production/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/technology/freespee.png?timestamp=d8f4ab6de90f4c4a7f3274feedf7032c"
+website_link: "http://www.freespee.com/"
```
Comment (dtothefp) [inline website/partners/technology/freespee/index.hbs]: @kellyfinn all `http` and `https` prefixes for links should be removed

**Comment by dtothefp** [comment]
LGTM

**Pull request merged**